Doc/library/asyncio-task.rst

```diff
@@ -282,6 +282,23 @@ Creating Tasks
       Added the *context* parameter.
 
 
+Task Cancellation
+=================
+
+Tasks can easily and safely be cancelled.
+When a task is cancelled, asyncio will raise a :exc:`asyncio.CancelledError`
+into the task at the next opportunity.
+
+Coroutines are recommended to use ``try/finally`` blocks to robustly
```

Comment:
The grammar here gets my hackles up a bit but I'm at a loss for how to put it better. (@CAM-Gerlach ?)

Reply:
Yeah, not only that but it is also potentially ambiguous (at least to a relative asyncio n00b like myself), as "coroutines" can be parsed either as the object of the sentence (i.e., that they are the target of the recommendation, which I'm guessing is the intended meaning), or that they are recommended _in order to use_ ``try/finally`` blocks.

Assuming the former, what about just

```suggestion
It is recommended that coroutines use ``try/finally`` blocks to robustly
```

Sure, it uses passive voice, but IMO its a good deal clearer